Makefile

```diff
@@ -3,15 +3,22 @@ SPHINXOPTS    ?= -W --keep-going
 SOURCEDIR     = spec
 BUILDDIR      = _site
 
-.PHONY: default clean build
+.PHONY: default install clean draft spec
 
-default: clean build
+default: install clean spec
+
+install:
+	pip install -e .[doc]
```

Comment:
Please don't do this. I would not expect `make` to go installing things in my current environment. It's possible to make the docs build without installing anything.